ndc-reference/tests/query/predicate_with_unrelated_exists_and_relationship/request.json

```diff
@@ -38,43 +38,22 @@
                     "predicate": {
                         "type": "exists",
                         "in_collection": {
-                            "type": "unrelated",
+                            "type": "related",
```

Comment:
This test might need a renaming given that it doesn't actually do any unrelated exists stuff anymore.

Reply:
Fixed, thanks!